### PR TITLE
Added index_reduce

### PR DESCRIPTION
**Pull request by Bright-in**
#14106

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def absolute(x):
     return ivy.abs(x)
 
 
-abs = absolute
+#abs = absolute
```
Comment (pillarxyz) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:22]: why did you comment this out ?

Comment (Bright-in) [reply ivy/functional/frontends/jax/numpy/mathematical_functions.py:22]: Hi sir, it appeared as part of the errors.

**Comment by Bright-in** [comment]
#14106

**Comment by Bright-in** [comment]
It has been updated

File: ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py [hunk 2]
```diff
@@ -176,6 +177,11 @@ def index_select(input, dim, index, *, out=None):
     return ivy.gather(input, index, axis=dim, out=out)
 
 
+@to_ivy_arrays_and_back
+def index_reduce(dim, index, source, reduce, *, include_self=True):
+    return torch.index_reduce(dim, index, source, reduce, include_self=include_self)
```
[the remark this answers is not in the document]
Comment (Bright-in) [reply ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py:182]: Can you please help me with this one. I tried using ivy.index_reduce but it said there's no such member in ivy.

[the remark this answers is not in the document]
Comment (Bright-in) [reply ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py]: I tried

**Comment by pillarxyz** [comment]
The function you used to implement the frontend doesn't exist I think

File: ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py [hunk 3]
```diff
@@ -176,6 +177,11 @@ def index_select(input, dim, index, *, out=None):
     return ivy.gather(input, index, axis=dim, out=out)
 
 
+@to_ivy_arrays_and_back
+def index_reduce(dim, index, source, reduce, *, include_self=True):
+    return ivy.index_reduce_axis(dim, index, source, reduce, include_self=include_self)
```
Comment (pillarxyz) [inline ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py:182]: this one right here, you really need to implement the function in the right way

**Pull request closed without merging**